stdlib/public/Platform/TiocConstants.swift

```diff
@@ -0,0 +1,131 @@
+// TIOC constants, needed only on Darwin and FreeBSD
+
+#if os(OSX) || os(iOS) || os(tvOS) || os(watchOS) || os(FreeBSD)
+
+/// get modem control state
+public let TIOCMODG:UInt = 0x40047403
+/// set modem control state
+public let TIOCMODS:UInt = 0x80047404
+/// set exclusive use of tty
+public let TIOCEXCL:UInt = 0x2000740d
+/// reset exclusive use of tty
+public let TIOCNXCL:UInt = 0x2000740e
+/// flush buffers
+public let TIOCFLUSH:UInt = 0x80047410
+/// get termios struct
+public let TIOCGETA:UInt = 0x40487413
+/// set termios struct
+public let TIOCSETA:UInt = 0x80487414
+/// drain output, set
+public let TIOCSETAW:UInt = 0x80487415
+/// drn out, fls in, set
+public let TIOCSETAF:UInt = 0x80487416
+/// get line discipline
```

Comment:
They come from macOS ttycom.h and iocom-compact.h headers, that are under APSL+BSD, not actual code but i guess the original license should be included anyway. Since there is no other APSL code in the project, should i just remove these comments?